Xamarin.Forms.Platform.Android/Extensions/ImageViewExtensions.cs

```diff
@@ -33,12 +32,15 @@ internal static class ImageViewExtensions
 			if (newImageSource != null && Equals(previousImageSource, newImageSource))
 				return;
 
-			imageController?.SetIsLoading(true);
```

Comment:
Did you mean to move this?

Reply:
Yes, I moved this to line 42. In my head, I think that we should `SetIsLoading(true)` after we set the LoadingPlaceholder. If you don't like this idea, I can move this call to the top

Reply:
I think it makes sense to still be here. 

Placeholder => Source => Error 

I think are all part of the same Loading operation